src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +13,12 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
+        User foundUser = userService.findByEmail(email);
+        if (foundUser != null && foundUser.getPassword().equals(password)) {
+            return true;
+        }
         return false;
```

Comment:
try to inline it

Reply:
No problems, changed. But if statement looks a little bit readable